src/e2e-test/features/gcs/source/GCSToGCSAdditonalTests.feature

```diff
@@ -266,10 +266,12 @@ Feature: GCS source - Verification of GCS to GCS Additional Tests successful
     Then Deploy the pipeline
     Then Run the Pipeline in Runtime
     Then Wait till pipeline is in running state
+    Then Open and capture logs
     And Verify the pipeline status is "Failed"
+    Then Close the pipeline logs
     Then Open Pipeline logs and verify Log entries having below listed Level and Message:
       | Level | Message                                                     |
-      | ERROR | errorMessageMultipleFileWithFirstRowAsHeaderEnabled         |
+      | WARN | errorMessageMultipleFileWithFirstRowAsHeaderEnabled         |
```

Comment:
how is this `WARN`?

I can see the level is `ERROR`: 
![image](https://github.com/user-attachments/assets/14edf78a-82d8-4585-a159-ba2c8752639c)

How is this test passing right now?

Reply:
Basically to make it compatible with both cdap and CDF we use warn here because in cdap it only show **NumberFormatException** error in only **WARN** level in advanced logs. You are checking it in raw logs. But the above mentioned step only checks in advanced logs section.

Reply:
Can we modify the step to check in raw logs? So that it is consistent in both cdap and CDF.